src/mame/drivers/bnstars.cpp

```diff
@@ -844,13 +844,13 @@ void bnstars_state::bnstars_sound_map(address_map &map)
 void bnstars_state::bnstars(machine_config &config)
 {
 	/* basic machine hardware */
-	V70(config, m_maincpu, 20000000); // 20MHz
+	V70(config, m_maincpu, 40_MHz_XTAL/2); // 20MHz
```

Comment:
Is 20 MHz correct in the comment?

Reply:
V70 clock is divided 40 MHz XTAL into 2; it results 20MHz.

Reply:
understood, thanks!